### ISSUE TITLE
NVDA is unable to read part of sentence if there is html tag nesting in Chrome Browser

### ISSUE THREAD

**Issue by dinesh4735** [nvaccess/nvda#12744]
### Steps to reproduce:
1. Add a html tag with in an existing HTML statement in your web page.  example: 
```html
<p>This is just a test statement<sup>nested statement</sup>rest of the sentence will be ignored</p>
```
2. Now run NVDA screen reader in chrome browser and tab or mouse over to the testing sentence

### Actual behavior:
1. The NVDA screen reader will read only the part of sentence and ignoring the remain sentence in chrome browser

### Expected behavior:
1. NVDA screen reader should read the complete statement as "This is just a test statement nested statement rest of the sentence will be ignored"

#### NVDA installed/portable/running from source:
NVDA client installed in a Windows Laptop
#### NVDA version:
2021.1
#### Windows version:
Windows10 Enterprise
#### Name and version of other software in use when reproducing the issue:
Chrome Version 92.0.4515.131 (Official Build) (64-bit)
#### Other information about your system:
Windows 
#### Does the issue still occur after restarting your computer?
Yes it still persists
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
Yes before upgrading to 2021.1 version, the behaviour was same
#### If add-ons are disabled, is your problem still occurring?
Problem still occurs
#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
Haven't tried and not sure how to run the COM Registration fixing tool

**Comment by feerrenrut** [comment]
@dinesh4735 welcome to the NVDA project.

Please note that GitHub issues (on the NVDA project) are intended for a technical audience, willing to investigate, test, and debug the project. If you are looking for end user support, please consider the [options to get help on the NV Access website](https://www.nvaccess.org/get-help/).

Unfortunately this issue does not adequately describe the situation, and we will not be able to investigate further. I'm not able to reproduce the issue as described.

> The NVDA screen reader will read only the part of sentence and ignoring the remain sentence in chrome browser

Which part of the sentence, please be specific.

What I tried:
Using the following sample:
```html
<button>before</button>
<p>This is just a test statement<sup>nested statement</sup>rest of the sentence will be ignored</p>
<button>after</button>
```
With NVDA 2021.1 in browse mode, focus the "before button", press down arrow. Announced: "This is just a test statement nested statement rest of the sentence will be ignored"

Reset NVDA configuration to factory defaults, same result.

I'm going to close this issue, however if more information is provided I will re-open.
